app/templates/crates.hbs

```diff
@@ -12,7 +14,6 @@
             {{ this }}
         {{/link-to}}
     {{/each}}
-    {{view Ember.Select content=alphabet value=letter}}
```

Comment:
This is used by the "mobile site" which you can see if you shrink the page by a good bit, so this needs to stay